examples/lead-generation/src/airtop-client.ts

```diff
@@ -0,0 +1,18 @@
+import { AirtopClient } from "@airtop/sdk";
+
+let airtopClientInstance: AirtopClient | null = null;
+
+/**
+ * Get an AirtopClient instance using the singleton pattern.
+ * 🔥 NOTE: This pattern only allows one instance with one API key.
+ * @param apiKey - The API key for the AirtopClient.
+ * @returns An instance of AirtopClient.
+ */
+export const getAirtopClient = (apiKey: string): AirtopClient => {
```

Comment:
this is a problem / security hole if this is what the hosted app executes because it means we'll use the api key of the first person who sets it